packages/ipfs-http-client/src/pubsub/subscribe.js

```diff
@@ -4,6 +4,9 @@ const bs58 = require('bs58')
 const { Buffer } = require('buffer')
 const log = require('debug')('ipfs-http-client:pubsub:subscribe')
 const SubscriptionTracker = require('./subscription-tracker')
+
+// TODO: Update streamToAsyncIterator with any chances in light of
+// this feature branch
 const { streamToAsyncIterator, ndjson } = require('../lib/core')
```

Comment:
@hugomrdias Can you give a short description of the new approach with `streamToAsyncIterator` compared with `toIterable`? Thanks!